tests/system/test_cluster/test_agent_groups/test_agent_groups_forced_change.py

```diff
@@ -0,0 +1,180 @@
+            type: fixture
+            brief: Reset the wazuh log files at the start of the test. Remove all registered agents from master.
+        - test_infra_managers
+            type: List
+            brief: list of manager hosts in enviroment
+        - test_infra_managers
+            type: List
+            brief: list of agent hosts in enviroment
+        - host_manager
+            type: HostManager object
+            brief: handles connection the enviroment's hosts.
+    assertions:
+        - Verify that after registering the agents appear as active in all nodes.
+        - Verify that after registering and after starting the agent, the indicated group is synchronized.
+        - Verify that after after adding a new node to the cluster, the agent's group data is synchronized.
+    expected_output:
+        - The 'Agent_name' with ID 'Agent_id' belongs to groups: 'group_name'.
+    '''
+    for group in agent_groups:
+        create_new_agent_group(test_infra_managers[0], group, host_manager)
+
+    agent1_data = register_agent(test_infra_agents[0], agent_host, host_manager, agent_groups[0])
+    agent2_data = register_agent(test_infra_agents[1], agent_host, host_manager, agent_groups[1])
+    agent3_data = register_agent(test_infra_agents[2], agent_host, host_manager, agent_groups[2])
+
+    restart_cluster(test_infra_agents, host_manager)
+    time.sleep(10)
+
+    # Check agent status in all nodes
+    check_agent_status(agent1_data[1], agent1_data[2], agent1_data[0], AGENT_STATUS_ACTIVE, host_manager, test_infra_managers)
+    check_agent_status(agent2_data[1], agent2_data[2], agent2_data[0], AGENT_STATUS_ACTIVE, host_manager, test_infra_managers)
+    check_agent_status(agent3_data[1], agent3_data[2], agent3_data[0], AGENT_STATUS_ACTIVE, host_manager, test_infra_managers)
+
+    # Check that agent has the expected group assigned in all nodes
+    check_agent_groups(agent1_data[1], agent_groups[0], test_infra_managers, host_manager)
+    check_agent_groups(agent2_data[1], agent_groups[1], test_infra_managers, host_manager)
+    check_agent_groups(agent3_data[1], agent_groups[2], test_infra_managers, host_manager)
+
+    # Force group change on wazuh-agent1
+    change_agent_group_with_wdb(agent1_data[1], agent_groups[1], test_infra_managers[0], host_manager)
+
+    # Check that agent has group set to Group2 in new node
+    check_agent_groups(agent1_data[1], agent_groups[1], test_infra_managers, host_manager)
+
+
+# Tests
+@pytest.mark.parametrize("test_infra_managers",[test_infra_managers])
+@pytest.mark.parametrize("test_infra_agents",[test_infra_agents])
+@pytest.mark.parametrize("host_manager",[host_manager])
+def test_force_group_change_during_sync(clean_environment, test_infra_managers, test_infra_agents, host_manager):
+    '''
+    description: Check that having a series of agents assigned with different groups, when an new node is added to
+    the cluster, the group data is synchronized to the new node.
```

Comment:
```suggestion
    description: Check that having a series of agents assigned with different groups, when an new node is added from the worker node, the group data is synchronized to the new node.
```

Reply:
The description is confused and has a lot of similarities with the previous test. Please check it.